Source/Core/Core/DSP/Interpreter/DSPIntTables.cpp

```diff
@@ -266,7 +265,7 @@ void InitInstructionTables()
   // ext op table
   for (size_t i = 0; i < s_ext_op_table.size(); i++)
   {
-    s_ext_op_table[i] = nop;
+    s_ext_op_table[i] = &Interpreter::nop_ext;
```

Comment:
It looks like the nop implementation is serving double duty: an actual handler for nop and an unrecognized opcode handler that reports an error message. Interpreter::nop() does both, but nop_ext() only does the former. By switching implementations to nop_ext(), this change eliminates the reporting of unrecognized opcodes.

Defining a dedicated unrecognized opcode handler would retain the intent and behavior of the original code.